repository/e.json

```diff
@@ -1431,6 +1431,19 @@
 				}
 			]
 		},
+		{
+			"name": "Exact Quick Find",
+			"details": "https://github.com/aafulei/sublime-exact-quick-find",
+			"author": "Aaron Fu Lei",
+			"labels": ["search", "text navigation", "text selection"],
+			"releases": [
+				{
+					"platforms": "*",
```

Comment:
If you support all platforms you can simply not specify it.

Reply:
The line `"platforms": "*",` has been removed. Thanks.